includes/class-sensei.php

```diff
@@ -1102,6 +1104,32 @@ public function wp_quicklatex_support() {
 		$this->maybe_add_latex_support_via( 'quicklatex_parser' );
 	}
 
+	/**
+	 * Overrides the shortcodes that Jetpack loads.
+	 *
+	 * Removes Vimeo and Youtube from the Jetpack shortcodes module to avoid iframes being converted
+	 * to shortcodes.
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param array $shortcodes Array of shortcodes to include.
+	 * @return array Revised array of shortcodes to include.
+	 */
+	public function remove_jetpack_shortcodes( $shortcodes ) {
+		$jetpack_shortcodes_dir = WP_CONTENT_DIR . '/plugins/jetpack/modules/shortcodes/';
+		$shortcodes_to_unload   = array( 'vimeo.php', 'youtube.php' );
+
+		foreach ( $shortcodes_to_unload as $shortcode ) {
+			$key = array_search( $jetpack_shortcodes_dir . $shortcode, $shortcodes, true );
```

Comment:
This works well and looks like it is the snippet on Jetpack's documentation, but `remove_jetpack_shortcodes` also passes the basename as the array key. Right now, it is dependent on (most cases, obviously) Jetpack being installed in `plugins/jetpack`. With the array key being the basename, I think we could just `unset( $shortcodes['youtube'] );`.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2426/commits/713a444ce086bfca620d77914673c3457acf74ac.